src/main.dart

```diff
@@ -1,8 +1,138 @@
 // TODO: Run task list app
+
+import 'dart:io';
+import 'task.dart';
+
 void main(List<String> args) {
-  print('Hello World');
+  final TaskListApp taskApp = TaskListApp();
+
+  print('------------------ Task Manager ------------------');
+  print('------ Welcome to our beautfuil Task Manager -----');
+  print('------  Please enter the menu option below  ------');
+  print('--------------------------------------------------');
+
+  print('--- 1 for "List Task"');
+  print('--- 2 for "Add Task"');
+  print('--- 3 for "Edit Task"');
+  print('--- 4 for "Delete Task"');
+  print('--- -1 for "Exit App"');
```

Comment:
Make this into a print function and maybe redisplay the menu every time after an operation has been executed